src/main/java/duke/task/Todo.java

```diff
@@ -0,0 +1,19 @@
+package duke.task;
+
+import duke.task.Task;
+
+public class Todo extends Task {
+
+  public Todo(String description, boolean status) {
```

Comment:
Perhaps a change to the argument name to sound more like a boolean? I noticed this issue occurring in classes extending task.

```suggestion
  public Todo(String description, boolean isMarked) {
```